### PR TITLE
[POU-91] carbon widget design

### PR DESCRIPTION
**Pull request by crimson-med**
AB#POU-91

# Why

To display the world map and the carbon reduction widget

# What

Overmind refactor for better stage management

Overmind Mocking for tests

Functional highlight of map on hover and dropdown hover

Functional data filtering (map and widget) based on: Region, Country, Timeframe

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -39,3 +39,5 @@ yarn.lock
 
 utils/dist
 utils/node_modules
+test-report.xml
```
Comment (morphet81) [inline .gitignore:42]: This one does not exist anymore

File: src/@types/State.ts
Comment (morphet81) [inline src/@types/State.ts:1]: Can you check that the naming is aligned with what we have on backend?

File: src/@types/Timeframe.ts
Comment (morphet81) [inline src/@types/Timeframe.ts:1]: Seems like it's the same said in 2 different ways, appart from six months and year to date.

File: src/app/layout.tsx [hunk 2]
```diff
@@ -2,33 +2,56 @@
...
+import '../i18n'
+import './globals.css'
 
 const overmind = createOvermind(config)
-
-// eslint-disable-next-line @typescript-eslint/quotes
+/*eslint @typescript-eslint/quotes: off*/
 const inter = Inter({ subsets: ['latin'] })
 
 const theme = extendTheme({
```
Comment (morphet81) [inline src/app/layout.tsx:23]: Can we put that in a different file?

File: src/@types/svg.ts
Comment (morphet81) [inline src/@types/svg.ts:1]: Comment

**Review by morphet81 — DISMISSED** [review]

**Review by MingLiang-PALOIT — APPROVED** [review]

**Pull request closed without merging**